packages/ui/Repeater/tests/Repeater-specs.js

```diff
@@ -9,156 +11,125 @@ describe('Repeater Specs', () => {
 	const CustomType = (props) => <div>{props.content}</div>;
 
 	test('should have a root span element', () => {
-		const subject = shallow(
-			<RepeaterBase childComponent="div">{stringItems}</RepeaterBase>
-		);
+		render(<RepeaterBase childComponent="div" data-testid="repeater">{stringItems}</RepeaterBase>);
 
 		const expected = 'span';
-		const actual = subject.first().name();
+		const rootElementName = screen.getByTestId('repeater').nodeName.toLowerCase();
```

Comment:
I guess.. we could change the expected to 'SPAN' instead of using `toLowerCase`.